mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
+  // let isHappy = true;
```

Comment:
You are absolutely correct to remove this line, you commented it out rather than deleting it, this is a smart thing to do when working on a problem.

Unfortunately, we discourage pushing work with comments like this once we are happy with the final solution, it's fine for work in progress but when you're ready to submit your final version we should always remove commented out code. There are great reasons for this, I'd be happy to share in our team channel, but if you'd like I can share this documentation from CYF: https://syllabus.codeyourfuture.io/guides/code-style-guide#dont-leave-lots-of-commented-out-code